From 404604eedb2882481bac172846c85661cbdf56d1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tom=C3=A1=C5=A1=20Macholda?= <tomas.macholda@nic.cz>
Date: Mon, 16 Mar 2026 12:01:11 +0100
Subject: [PATCH 1/2] gpio-button-hotplug: rename driver and compatible

driver name: gpio-button-hotplug{,-polled}
compatible: openwrt,gpio-button-hotplug{,-polled}
---
 .../gpio-button-hotplug/src/gpio-button-hotplug.c      | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/package/kernel/gpio-button-hotplug/src/gpio-button-hotplug.c b/package/kernel/gpio-button-hotplug/src/gpio-button-hotplug.c
index 9876dee90f..7cc1b59f44 100644
--- a/package/kernel/gpio-button-hotplug/src/gpio-button-hotplug.c
+++ b/package/kernel/gpio-button-hotplug/src/gpio-button-hotplug.c
@@ -32,7 +32,7 @@
 
 #define BH_SKB_SIZE	2048
 
-#define DRV_NAME	"gpio-keys"
+#define DRV_NAME	"gpio-button-hotplug"
 #define PFX	DRV_NAME ": "
 
 struct bh_event {
@@ -417,13 +417,13 @@ gpio_keys_get_devtree_pdata(struct device *dev)
 }
 
 static struct of_device_id gpio_keys_of_match[] = {
-	{ .compatible = "gpio-keys", },
+	{ .compatible = "openwrt,gpio-button-hotplug", },
 	{ },
 };
 MODULE_DEVICE_TABLE(of, gpio_keys_of_match);
 
 static struct of_device_id gpio_keys_polled_of_match[] = {
-	{ .compatible = "gpio-keys-polled", },
+	{ .compatible = "openwrt,gpio-button-hotplug-polled", },
 	{ },
 };
 MODULE_DEVICE_TABLE(of, gpio_keys_polled_of_match);
@@ -692,7 +692,7 @@ static struct platform_driver gpio_keys_driver = {
 	.probe	= gpio_keys_probe,
 	.remove	= gpio_keys_remove,
 	.driver	= {
-		.name	= "gpio-keys",
+		.name	= "gpio-button-hotplug",
 		.of_match_table = of_match_ptr(gpio_keys_of_match),
 	},
 };
@@ -701,7 +701,7 @@ static struct platform_driver gpio_keys_polled_driver = {
 	.probe	= gpio_keys_polled_probe,
 	.remove	= gpio_keys_remove,
 	.driver	= {
-		.name	= "gpio-keys-polled",
+		.name	= "gpio-button-hotplug-polled",
 		.of_match_table = of_match_ptr(gpio_keys_polled_of_match),
 	},
 };
-- 
2.47.3

